microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md

```diff
@@ -0,0 +1,191 @@
+
+![An ASP.NET project, opening Edge DevTools.](../../media/2022/03/devtools-extension-v-s-web-forms-designer.png)
+
+Edge DevTools for Visual Studio opens, with the **Elements** tool selected:
+
+![Microsoft Edge Developer Tools for Visual Studio: Elements tool of DevTools.](../../media/2022/03/devtools-extension-visual-studio-elements.png)
+
+The **Network** tool is also open by default:
+
+![Microsoft Edge Developer Tools for Visual Studio: Network tool of DevTools.](../../media/2022/03/devtools-extension-visual-studio-network.png)
+
+The **Inspect tool** (![Inspect tool icon.](../../media/2022/03/v-s-edge-devtools-inspect-tool-icon.png)) and **Toggle screencast** (![Toggle screencast icon.](../../media/2022/03/v-s-edge-devtools-toggle-screencast-icon.png)) are available, and the **More Tools** (![More Tools icon.](../../media/2022/03/more-tools-v-s-icon.png)) menu offers the [Issues](../../../issues/index.md), [Network conditions](../../../network-conditions/network-conditions-tool.md), and [Network request blocking](../../../network-request-blocking/network-request-blocking-tool.md) tools.
+
+See also:
+
+* [Edge Developer Tools for Visual Studio (Preview)](https://devblogs.microsoft.com/visualstudio/?p=237066&preview=1&_ppp=7aa7aef54f), at Visual Studio DevBlogs.
+* [Edge DevTools extension for Visual Studio](../../../../visual-studio/index.md#edge-devtools-extension-for-visual-studio).
+
+
+<!-- ====================================================================== -->
+## Layers in 3D View
+
+<!-- Title: Layers in 3D View -->
+<!-- Subtitle: The Layers tool isn't going away - find it in the 3D View tool today. -->
+
+Since [Microsoft Edge 88](../../2020/11/devtools.md#composited-layers-are-now-in-3d-view), the **3D View** tool has provided an improved version of the **Layers** tool.  Now, in Microsoft Edge 99, the standalone **Layers** tool has been removed, and contains a link that redirects you to the **Composited Layers** view within the **3D View** tool.  You can find all the functionality from the **Layers** tool in the **3D View** tool.
+
+![The Layers tool now links to the 3D View tool with the Composited Layers view open.](../../media/2022/03/layers-3d-view-tool.png)
+<!-- work item > layersmove.gif -->
+
+See [Navigate z-index, DOM, and layers using the 3D View tool](../../../3d-view/index.md).
+
+
+<!-- ====================================================================== -->
+## Use your source maps to unminify performance profiles
+
+<!-- Title: The Performance tool can now display unminified call stacks -->
+<!-- Subtitle: Use the new Unminify button in the Performance tool to download an unminified version of the performance profile you recorded. -->
+
+In previous versions of Microsoft Edge, recording a performance profile in the Performance tool would produce a minified call stack. The call stack was not using source maps, even if you hosted them on your server, to unminify the call stack.
```

Comment:
```suggestion
In previous versions of Microsoft Edge, recording a performance profile in the Performance tool would produce a minified call stack. The call stack did not use source maps to unminify the call stack, even if you hosted the source maps on your server.
```

Reply:
@zoherghadyali - please review my suggestion.

Reply:
Suggestion v3:

Recording a performance profile in the **Performance** tool produces a minified call stack.  In previous versions of Microsoft Edge, the call stack (flame chart) didn't use source maps to display your original function names, even if you hosted the source maps on your server.

Reply:
To match with the docs for this feature, let's just do: 

Recording a performance profile in the Performance tool produces a minified flame chart. In previous versions of Microsoft Edge, the flame chart didn't use source maps to display your original function names, even if you hosted the source maps on your server.